lib/tickle/repeater.rb

```diff
@@ -22,28 +22,29 @@ def self.scan_for_numbers(token)
   end
 
   def self.scan_for_ordinal_names(token)
-    scanner = {/first/ => '1st',
-      /second/ => '2nd',
-      /third/ => '3rd',
-      /fourth/ => '4th',
-      /fifth/ => '5th',
-      /sixth/ => '6th',
-      /seventh/ => '7th',
-      /eighth/ => '8th',
-      /ninth/ => '9th',
-      /tenth/ => '10th',
-      /eleventh/ => '11th',
-      /twelfth/ => '12th',
-      /thirteenth/ => '13th',
-      /fourteenth/ => '14th',
-      /fifteenth/ => '15th',
-      /sixteenth/ => '16th',
-      /seventeenth/ => '17th',
-      /eighteenth/ => '18th',
-      /nineteenth/ => '19th',
-      /twentieth/ => '20th',
-      /thirtieth/ => '30th',
-    }
+    scanner = { 
+                /first/       => '1st',
+                /second/      => '2nd',
+                /third/       => '3rd',
+                /fourth/      => '4th',
+                /fifth/       => '5th',
+                /sixth/       => '6th',
+                /seventh/     => '7th',
+                /eighth/      => '8th',
+                /ninth/       => '9th',
+                /tenth/       => '10th',
+                /eleventh/    => '11th',
+                /twelfth/     => '12th',
+                /thirteenth/  => '13th',
+                /fourteenth/  => '14th',
+                /fifteenth/   => '15th',
+                /sixteenth/   => '16th',
+                /seventeenth/ => '17th',
+                /eighteenth/  => '18th',
+                /nineteenth/  => '19th',
+                /twentieth/   => '20th',
+                /thirtieth/   => '30th',
+              }
```

Comment:
This sort of refactoring is personal preference... but sometimes a little bit of formatting is enough to increase confidence in the code.